Opentelemetry error during ert run

While running `drogon_design.ert` on the current testing release I've got the following error:
```sh
2025-02-03 08:42:14,400 [ERROR] azure.monitor.opentelemetry.exporter.export._base: Data drop 400: 103: Field 'time' on type 'Envelope' is older than the allowed min date. Expected: now - 172800000ms {'additional_properties': {}, 'version': 1, 'name': 'Microsoft.ApplicationInsights.RemoteDependency', 'time': '2025-01-29T15:02:12.872982Z', 'sample_rate': 100, 'sequence': None, 'instrumentation_key': , 'tags': {<ContextTagKeys.AI_DEVICE_ID: 'ai.device.id'>: '', <ContextTagKeys.AI_DEVICE_LOCALE: 'ai.device.locale'>: 'en_US', <ContextTagKeys.AI_DEVICE_OS_VERSION: 'ai.device.osVersion'>: '#1 SMP Mon Dec 16 04:25:43 EST 2024', <ContextTagKeys.AI_DEVICE_TYPE: 'ai.device.type'>: 'Other', <ContextTagKeys.AI_INTERNAL_SDK_VERSION: 'ai.internal.sdkVersion'>: 'ulm_py3.11.11:otel1.29.0:ext1.0.0b32', <ContextTagKeys.AI_CLOUD_ROLE: 'ai.cloud.role'>: 'ert', <ContextTagKeys.AI_CLOUD_ROLE_INSTANCE: 'ai.cloud.roleInstance'>: '', <ContextTagKeys.AI_INTERNAL_NODE_NAME: 'ai.internal.nodeName'>: '', <ContextTagKeys.AI_OPERATION_ID: 'ai.operation.id'>: '552c364b281f3cefacd445a49020ae6b', <ContextTagKeys.AI_OPERATION_PARENT_ID: 'ai.operation.parentId'>: 'b9d1094516f16d09'}, 'data': <azure.monitor.opentelemetry.exporter._generated.models._models_py3.MonitorBase object at>}.
```

--- a/src/ert/logging/__init__.py
+++ b/src/ert/logging/__init__.py
@@ -2,6 +2,7 @@ import logging
 import os
 import pathlib
 import sys
+from collections.abc import Callable
 from datetime import datetime
 from types import TracebackType
 from typing import Any
@@ -74,3 +75,15 @@ class TerminalFormatter(logging.Formatter):
         | tuple[None, None, None],
     ) -> str:
         return ""
+
+
+def suppresse_logs(logs_to_suppress: list[str]) -> Callable[[logging.LogRecord], bool]:
+    """Suppresses logs from loggers listed in logs_to_suppress"""
+
+    def filter(record: logging.LogRecord) -> bool:
+        for log_name in logs_to_suppress:
+            if record.name.startswith(log_name):
+                return False
+        return True
+
+    return filter

--- a/src/ert/logging/storage_log.conf
+++ b/src/ert/logging/storage_log.conf
@@ -5,12 +5,17 @@ formatters:
     format: '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
   info:
     format: '%(message)s'
+filters:
+  suppress_not_user_relevant:
+    (): ert.logging.suppresse_logs
+    logs_to_suppress: [azure.monitor]
 handlers:
   default:
     level: WARNING
     formatter: standard
     class: logging.StreamHandler
     stream: ext://sys.stdout
+    filters: [suppress_not_user_relevant]
   infohandler:
     level: INFO
     formatter: info

